testcases.py

```diff
@@ -1114,7 +1114,7 @@ def _check_ecn_any(self, e) -> bool:
 
     def _check_ecn_marks(self, e) -> bool:
         return (
-            e[ECN.NONE] == 0
+            e[ECN.NONE] >= 0
```

Comment:
I assume `e[ECN.NONE]` is never `<0`, thus might as well remove the condition all together, or, for documentation purposes, leave as is, but explain why it is `>=` and not `==`.